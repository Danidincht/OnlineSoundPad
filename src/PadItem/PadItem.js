function PadItem(props) {
	return(
	<div>
		{props.title}
		<audio
			src={props.audio}
			controls={true}
		></audio>
	</div>
	);
}

export default PadItem;