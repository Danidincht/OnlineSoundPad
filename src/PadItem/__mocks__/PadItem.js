function PadItem() {
	return (
		<div>PadItemMockComponent</div>
	);
}

export default PadItem;
