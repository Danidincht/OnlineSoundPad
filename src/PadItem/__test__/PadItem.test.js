import { render, screen } from "@testing-library/react";
import PadItem from '../PadItem';

it('should render the PadItem component', () => {
	// Given -  When
	const page = render(<PadItem />).baseElement;

	// Then
	expect(page).toMatchSnapshot();
});