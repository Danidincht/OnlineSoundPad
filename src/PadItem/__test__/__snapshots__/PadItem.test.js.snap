// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the PadItem component 1`] = `
<body>
  <div>
    <div>
      PadItem title
      <audio
        controls=""
      />
    </div>
  </div>
</body>
`;
